lib/components/artist/biography.js

```diff
@@ -29,13 +29,13 @@ class Biography extends React.Component {
       <View style={{marginLeft: sideMargin, marginRight: sideMargin}}>
         <Headline style={{ marginBottom: 20 }}>Biography</Headline>
         { this.blurb(artist) }
-        <SerifText style={{ marginBottom: 40 }} numberOfLines={0}>{this.bioText()}</SerifText>
+        <SerifText style={{ marginBottom: 40, fontSize: 16, lineHeight: 20 }} numberOfLines={0}>{this.bioText()}</SerifText>
       </View>
     );
   }
 
   blurb(artist) {
-    return artist.blurb ? <SerifText style={{ marginBottom: 20, lineHeight: 25 }} numberOfLines={0}>{removeMarkdown(artist.blurb)}</SerifText> : null;
+    return artist.blurb ? <SerifText style={{ marginBottom: 20, fontSize: 16, lineHeight: 20 }} numberOfLines={0}>{removeMarkdown(artist.blurb)}</SerifText> : null;
```

Comment:
Fixing. Also realised that I think there should be less space between the "Past Shows" header and the list, so will sort that too